hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/QueryUtil.java

```diff
@@ -49,7 +49,9 @@
 import java.util.List;
 import java.util.Map.Entry;
 
-final class QueryUtil {
+public final class QueryUtil {
+    public static final String DEFAULT_UNIQUE_KEY_TRANSFORMATION = "OBJECT";
```

Comment:
this change does not belong to this PR. BTW, this class contains 2 completely unrelated groups of static methods used only in one place outside this class: one for joins, other one for index scan.

Reply:
It solves a code duplication issue. I moved these constants to the appropriate utility class. If you have a better placement idea - you may suggest them.

Reply:
`com.hazelcast.sql.impl.extract.KEY`.

`DEFAULT_UNIQUE_KEY_TRANSFORMATION` can stay where it were but also it might use `UniqueKeyTransformation.OBJECT.name`.

Reply:
Fixed https://github.com/hazelcast/hazelcast/pull/25006/commits/7d32b9ca07ee9c8a83d7d19158969dac085c5ab1